ai21/clients/studio/resources/chat/async_chat_completions.py

```diff
@@ -23,7 +26,10 @@ async def create(
         top_p: float | NotGiven = NOT_GIVEN,
         stop: str | List[str] | NotGiven = NOT_GIVEN,
         n: int | NotGiven = NOT_GIVEN,
-        stream: Optional[Literal[False]] | NotGiven = NOT_GIVEN,
+        stream: Optional[False] | NotGiven = NOT_GIVEN,
+        tools: List[ToolDefinition] | NOT_GIVEN = NOT_GIVEN,
```

Comment:
Notice it needs to be `| NotGiven = NOT_GIVEN` and not `| NOT_GIVEN = NOT_GIVEN` since `NOT_GIVEN` is the value and not the type